### PR TITLE
emr create fabric new doc

### PR DESCRIPTION
**Pull request by meiprophecy**
emr fabric documentation

### CODE REVIEW

File: docs/architecture/deployment/emr.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Livy does not work with Python version 3.8 and above.
 
 Navigate to Amazon EMR service and create a cluster. Under **Application bundle** select **Custom**.
 
-Choose appropriate applications to include in your installation. At a minimum, please make sure sure **Livy** , **Spark** and and **Hive** are included in the install.
+Choose appropriate applications to include in your installation. At a minimum, please make sure sure **Livy** and **Spark** are included in the install.
```
Comment (atbida) [inline docs/architecture/deployment/emr.md:25]: Overall comment: 
The content of this page seems duplicated with the content on this page: 
http://localhost:3000/low-code-spark/fabrics/EMR-fabric-configuration

Line 38: 
Do we require all https ports, or 443 or 8443?
https://www.hostinger.com/tutorials/https-port

Line 61: 
Nitpick: this screenshot is slightly cutting off the text.

File: docs/low-code-spark/fabrics/emr-fabric.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Enter your AWS credentials under **Access Key** and **Secret Key**. Choose the **Region** that your EMR cluster is running in.
+
+**Click** on **Fetch environments**.
+<img src={require('./img/emr-aws-cred.png').default} alt="EMR cred" width="75%" />
+
+**Select** your EMR cluster that you would like to connect to under **Spark Environment**.
+
+<img src={require('./img/emr-cluster-select.png').default} alt="EMR select" width="75%" />
+
+Most of the fields should be automatically populated after selecting a EMR cluster. Enter the S3 path that points to the location where you would like your logs to persist.
```
Comment (atbida) [inline docs/low-code-spark/fabrics/emr-fabric.md:69]: Should we add a note about S3 bucket/path permissions? 
What is the entity that should be enabled to write logs?
EMR cluster - IAM role?
Prophecy cluster - IAM role? 
Individual users ?
Service principals?

File: docs/low-code-spark/fabrics/img/arch_customervpc.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/arch_customervpc.png:1]: I don't see where these architecture images are used

File: docs/low-code-spark/fabrics/img/complete-f-abric.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/complete-f-abric.png:1]: looks like a duplicate image

File: docs/low-code-spark/fabrics/img/configure-j-ob-size.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/configure-j-ob-size.png:1]: duplicate image

File: docs/low-code-spark/fabrics/img/connectivity-databricks.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/connectivity-databricks.png:1]: I don't see where this figure is used

File: docs/low-code-spark/fabrics/img/create-entity-emr-fabric.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/create-entity-emr-fabric.png:1]: duplicate image

File: docs/low-code-spark/fabrics/img/emr-f-abric.png
Comment (atbida) [inline docs/low-code-spark/fabrics/img/emr-f-abric.png:1]: duplicate image

**Review by atbida — CHANGES_REQUESTED** [review]
@meiprophecy looking great! Some comments: 

1. The text is duplicated in these two places. Do you want to keep the full instructions in one place, and refer to them from the other place?
docs/low-code-spark/fabrics/EMR-fabric-configuration
docs/architecture/deployment/EMR-livy-installation-guide

2. Looks like some of the architecture diagrams (very nicely assembled!) didn't get used?

3. Some duplicate diagrams. 

4. I added the word "network" on "Configure connectivity" headings.

File: docs/low-code-spark/fabrics/emr-fabric.md (hunk 2 above)
Comment (atbida) [reply docs/low-code-spark/fabrics/emr-fabric.md:69]: no changes needed; this is handled by the prophecy token

**Review by atbida — APPROVED** [review]
made the needed changes; looks good!

**Review by atbida — APPROVED** [review]

**Review by JKChhabra — APPROVED** [review]

**Pull request merged**